### PR TITLE
Add postsubmit/periodic to auto update the openshift-priv-admins group

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: ci-operator/jobs/openshift/config/openshift-config-master-postsubmits.yaml [hunk 1]
```diff
@@ -39,3 +39,27 @@ postsubmits:
...
+      oauth_token_secret:
+        key: oauth
+        name: oauth-token
+    name: branch-ci-openshift-config-master-group-update
+    run_if_changed: org/config.yaml
+    spec:
+      serviceAccountName: group-auto-updater
+      containers:
+      - args:
+        - --peribolos-config=org/config.yaml
```
Comment (hongkailiu) [inline ci-operator/jobs/openshift/config/openshift-config-master-postsubmits.yaml]: Where does `org/config.yaml` come?

Comment (droslean) [reply ci-operator/jobs/openshift/config/openshift-config-master-postsubmits.yaml]: from the `openshift/config` repo

**Comment by hongkailiu** [comment]
/lgtm

/hold

@droslean 
please cancel when it is ready.

**Comment by petr-muller** [comment]
LGTM

**Pull request merged**